base/system/winlogon/sas.c

```diff
@@ -1320,6 +1325,11 @@ RegisterHotKeys(
     Session->TaskManHotkey = RegisterHotKey(hwndSAS, HK_CTRL_SHIFT_ESC, MOD_CONTROL | MOD_SHIFT, VK_ESCAPE);
     if (!Session->TaskManHotkey)
         WARN("WL: Warning: Unable to register Ctrl+Alt+Esc hotkey!\n");
+    
+    RegisterHotKey(hwndSAS, HK_SNAP_LEFT, MOD_WIN, VK_LEFT);
```

Comment:
Not sure this should be done in Winlogon; any reason why?

Reply:
Why not? this is the place where global hotkeys for Control-Alt-Delete and Shift-Control-Esc are registered